scripts/build/artifactory.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+#to be executed from the root project directory
+git rev-parse --short HEAD > version.txt
```

Comment:
I think a more reliable solution would be to use full hash instead of its abbreviated version.